swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -10793,6 +11034,29 @@ components:
           - "booked"
           - "pending"
           - "both"
+          - "all"
+
+
+    bookingStatusCard:
+      name: bookingStatus
+      in: query
+      description: |
+        Permitted codes are 
+          * "booked",
+          * "pending",  
+          * "both",
+        "booked" shall be supported by the ASPSP.
+        To support the "pending" and "both" feature is optional for the ASPSP, 
+        Error code if not supported in the online banking frontend.
+        If supported, "both" means to request transaction reports of transaction of bookingStatus either "pending" or "booked".
+        
+      required: true
+      schema:
+        type: string
+        enum:
+          - "booked"
+          - "pending"
+          - "both"
```

Comment:
Relevant Change: bookingStatusCard renamed bookingStatus
Adoption approach: Follow the change